packages/portalnetwork/src/client/client.ts

```diff
@@ -49,7 +49,7 @@ export class PortalNetwork extends EventEmitter<PortalNetworkEvents> {
   metrics: PortalNetworkMetrics | undefined
   logger: Debugger
   ETH: ETH
-
+  enrCache: Map<string, ENR>
```

Comment:
```suggestion
  enrCache: Map<NodeId, ENR>
```

I know this is purely semantics but it's a nice one to explicitly define how we're mapping to the ENR.

You'll also need to add the import from `@chainsafe/enr`